### PR TITLE
[LEVWEB-1391] Add extra link to navbar

### PR DESCRIPTION
**Pull request by aarongill1**

### CODE REVIEW

**Review by daniel-ac-martin — COMMENTED** [review]
Could do with a test for that link text. (As you did that with the others.)

And it probably needs some sort of version bump.

**Comment by daniel-ac-martin** [comment]
Looks good otherwise though.

File: test/lev-top-nav/Index.test.js [hunk 1]
```diff
@@ -26,4 +26,9 @@ describe('component', () => {
     let link = root.findByProps({className: "page-link", href: '/history'});
     expect(link.props.children).toEqual('History');
   });
+
+  it('contains a link to the user activity report', () => {
+    let link = root.findByProps({className: "page-link", href: 'https://lev.homeoffice.gov.uk/audit/user-activity/'});
```
Comment (RobinKnipe) [inline test/lev-top-nav/Index.test.js:31]: @aarongill1 - I might be tempted to add the `target: '_blank'` prop in here too, as that is relatively important, but it's up to you...

**Review by RobinKnipe — APPROVED** [review]
1 small consideration, but ok to merge!

File: test/lev-top-nav/Index.test.js (hunk 1 above)
Comment (aarongill1) [reply test/lev-top-nav/Index.test.js]: Done

**Pull request merged**